MediaBrowser.Plugins.Anime/MediaBrowser.Plugins.Anime.csproj

```diff
@@ -1,23 +1,22 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;</TargetFrameworks>
+    <TargetFrameworks>netcoreapp2.1;</TargetFrameworks>
```

Comment:
Hmm, doesn't Jellyfin use netstandard2.0?

Reply:
I didn't get it to compile and it complained, so I basically had to do this. I'm open for suggestions on why it happened and how it can be avoided though.

Reply:
Jellyfin itself is netcore, supporting libs are netstandard, this should be netstandard too.

Reply:
Set back to netstandard2.0